pkg/cmd/target/view.go

```diff
@@ -35,6 +36,9 @@ func NewCmdView(f util.Factory, ioStreams util.IOStreams) *cobra.Command {
 
 	o.AddFlags(cmd.Flags())
 
+	f.TargetFlags().AddFlags(cmd.Flags())
+	flags.RegisterCompletionFuncsForTargetFlags(cmd, f, ioStreams, cmd.Flags())
```

Comment:
Not related to this PR but why do we pass the flagset to this function. It is not used. The flags must already be registered. And I think it was a very bad idea to name this package `flags` which always create a name conflict if you want to have variable named flags. I anyway think it should be part of the same package the TargetFlags belongs to.

Reply:
true, I have created an issue for it https://github.com/gardener/gardenctl-v2/issues/296